src/function/matrix/sqrtm.js

```diff
@@ -65,6 +65,7 @@ export const createSqrtm = /* #__PURE__ */ factory(name, dependencies, ({ typed,
   return typed(name, {
     'Array | Matrix': function (A) {
       const size = isMatrix(A) ? A.size() : arraySize(A)
+      console.log(A, size, size.length)
```

Comment:
Can you clean up this `console.log`?

Reply:
I'm a little ashamed that I forgot about that.
It's fixed now

Reply:
😂 happens to me too every now and then